app/views/studies/new.html.erb

```diff
@@ -16,8 +16,8 @@
   <p class="text-danger"><strong><i class="fas fa-fw fa-exclamation-triangle"></i> You cannot change the billing project of a workspace once it has been created.</strong></p>
   <br/>
   <h4>Required file format & ontologies</h4>
-  <p>Studies must have a metadata file specifying: <strong>species</strong>, <strong>disease</strong>, <strong>organ</strong>, <strong>library_preparation_protocol</strong>, <strong>sex</strong>, <strong>cell name</strong>, <strong>biosample_id</strong>, and <strong>donor_id</strong>.</p>  Metadata files must provide this information using <%= link_to 'SCP Conventional File Format', 'https://github.com/broadinstitute/single_cell_portal/wiki/Metadata-File#scp-conventional-file-format', target: :_blank %>. Select ontology terms from <%= link_to 'designated ontologies', 'https://github.com/broadinstitute/single_cell_portal/wiki/Metadata-File#Required-Conventional-Metadata', target: :_blank %> to adhere to <%= link_to 'metadata validation rules', 'https://github.com/broadinstitute/single_cell_portal/wiki/Metadata-File#metadata-validation-rules', target: :_blank %>.  These requirements make your study data discoverable through Single Cell Portal's Advanced Search feature.
-</div>
+  <p>Studies must have a metadata file specifying: <strong>species</strong>, <strong>disease</strong>, <strong>organ</strong>, <strong>library_preparation_protocol</strong>, <strong>sex</strong>, <strong>cell name</strong>, <strong>biosample_id</strong>, and <strong>donor_id</strong>.</p>  Metadata files must provide this information using <%= link_to 'SCP Conventional File Format', 'https://singlecell.zendesk.com/hc/en-us/articles/360061006411-Metadata-Convention', target: :_blank %>. Select ontology terms from <%= link_to 'designated ontologies', 'https://singlecell.zendesk.com/hc/en-us/articles/360060609852-Required-Metadata', target: :_blank %> to adhere to <%= link_to 'metadata validation rules', 'https://singlecell.zendesk.com/hc/en-us/articles/360060610012-Metadata-Validation', target: :_blank %>.  These requirements make your study data discoverable through Single Cell Portal's Advanced Search feature.
+  </div>
```

Comment:
I think this whitespace is unintentional?  it doesn't look like it aligns with the opening tag

Reply:
oops yes not intentional thanks!